README.md

```diff
@@ -19,28 +19,28 @@ of tests from the command-line. Concrete test runners are then simply plugged
 in, so they all work in the same unified way. Currently the following test
 runners are supported:
 
-| Language       | Test Runners                                                                  | Identifiers                                                                                     |
-| -------------: | :---------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------- |
-| **C#**         | .NET                                                                          | `xunit`, `dotnettest`                                                                           |
-| **Clojure**    | Fireplace.vim                                                                 | `fireplacetest`                                                                                 |
-| **Crystal**    | Crystal                                                                       | `crystalspec`                                                                                   |
-| **Elixir**     | ESpec, ExUnit                                                                 | `espec`, `exunit`                                                                               |
-| **Elm**        | elm-test                                                                      | `elmtest`                                                                                       |
-| **Erlang**     | CommonTest                                                                    | `commontest`                                                                                    |
-| **Go**         | Ginkgo, Go                                                                    | `ginkgo`, `gotest`                                                                              |
-| **Java**       | Maven, Gradle                                                                 | `maventest`, `gradletest`                                                                       |
```

Comment:
You'll need to rebase against master so it doesn't remove this

Reply:
Hi Mathew, thanks for reviewing this PR. I couldn't realize how to rebase commits already pushed to remote. So, I ended up doing two more commits to fix the Readme. Maybe there is an option to "squash" things before merging, like [this](https://help.github.com/articles/about-pull-request-merges/#squash-and-merge-your-pull-request-commits), on your side? If not, then I can delete my repository and fork again to apply the changes at once, so that we do not pollute the repository with so many commits. What would you suggest?